README.md

```diff
@@ -1,10 +1,17 @@
 # how to open fetch-project-experiment
+
 - In the root of the project, right click in the index.html file and select the option `open with live server`;
 
 # project content
-- This project it's not completed, I'm still working on it and it's an `experiment` of what I understood until from the variuos courses;
-- I'm trying to put toghether all the concepts I've understood until now, the JavaScript file is still on `work in progress mode`.
-- This project returns a random quote from the `github` API every time the `get` button is clicked;
-- On the left side there is a `navigation bar` with a button to open and close the bar.
-- On the right there is another menu bar which contains a `form`.
-- For now only the `get` button to display the `random quote` is working.
+
+- This project returns a random quote from the `quote garden` API every time the `get` button is clicked;
+- On the left side there is a `navigation bar` with a `button` to open and close the bar;
+- The `random quote` will be displayed in the `quote container`.
+- The links in the navigation bar are not linked to other pages;
+- On the right the `join us` button  if pressed opens a registration form;
```

Comment:
On the right their's a `join us` button, when pressed it opens a registration form;